packages/tailwindcss-x/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tailwindcss-x",
+  "private": true,
```

Comment:
I think we should add the description, author, license, homepage and similar fields like we do for other packages